staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -45,16 +45,19 @@ type ServerRunOptions struct {
 	MinRequestTimeout           int
 	TargetRAMMB                 int
 	WatchCacheSizes             []string
+
+	AdmissionPlugins *admission.Plugins
```

Comment:
@p0lyn0mial It's probably worth splitting out `AdmissionOptions` with this, along with `AdmissionControl` and `AdmissionControlConfigFile`.